docs/guides/devops-lifecycle/integrate-web-modeler-in-ci-cd.md

```diff
@@ -0,0 +1,332 @@
+- Process reviews differ from code reviews, occurring on visual diagrams rather than XML.
+
+![Sample CI/CD setup with Web Modeler](img/modeler-ci-cd.png)
+
+### Obtain API clients and tokens
+
+Before getting started, obtain API clients and tokens for integrating Web Modeler and accessing the process engine via API:
+
+- [Obtain an API token for Web Modeler](/apis-tools/web-modeler-api/index.md#authentication)
+- [Obtain an API client for Zeebe](/guides/setup-client-connection-credentials.md)
+
+### Disable manual deployments from Web Modeler
+
+To enforce pipeline-driven deployments to your environments, consider disabling manual deployments.
+
+<Tabs groupId="disableDeployments" defaultValue="sm" values={[{label: 'Self-Managed', value: 'sm', }, {label: 'SaaS', value: 'saas', },]} >
+<TabItem value="sm">
+
+Disable manual deployments for any user by configuring environment variables `ZEEBE_BPMN_DEPLOYMENT_ENABLED` and `ZEEBE_DMN_DEPLOYMENT_ENABLED` as documented [here](/self-managed/modeler/web-modeler/configuration/configuration.md/#general).
+
+</TabItem>
+<TabItem value="saas">
+
+Remove the **Developer** role from users in Console to restrict their deployment permissions. Read more in the [user roles documentation](/components/console/manage-organization/manage-users.md).
+
+</TabItem>
+</Tabs>
+
+### Triggering CI/CD
+
+You need triggers to initiate the pipeline for files or projects. Choose between manual pipeline start or automatic background triggers based on events. Common approaches include:
+
+- Initiating the pipeline manually from your CI/CD tool/platform by uploading the file intended for deployment.
+- Starting the CI pipeline by creating a pull/merge request in the version control system.
+- Triggering pipelines by listening to milestones with certain characteristics.
+
+#### Sync files with version control
+
+Synchronize files between Web Modeler and version control systems (VCS) and vice versa. Manage both files and projects by using a complete set of CRUD (create, read, update, delete) operations provided by the Web Modeler API. By syncing files from Web Modeler to your VCS, you benefit from full file ownership and avoid duplicated data housekeeping.
+
+For automatic file synchronization, consider maintaining a secondary system of record for mapping Web Modeler projects to VCS repositories. This system also monitors the project-to-repository mapping and update timestamps.
+
+<!-- To be added once GA released:
+
+To listen to changes in Web Modeler, you currently need to implement a polling approach that compares the update dates with the last sync dates recorded. Use the `POST /api/beta/files/search` [endpoint](https://modeler.cloud.camunda.io/swagger-ui/index.html#/Files/searchFiles) with this payload to identify recently updated files:
+
+```json title="POST /api/beta/files/search"
+{
+  "filter": {
+    "projectId": "<PROJECT TO SYNC>",
+    "updated": "<LAST SYNC DATE>",
+  ],
+  "page": 0,
+  "size": 50
+}
+```
+-->
```

Comment:
@christian-konrad As https://github.com/camunda/web-modeler/issues/5754 is now merged filtering by date ranges is now possible. Documentation is still pending but it basically works the same way as for the other APIs by appending a date range suffix like `||/m` to the date string, see also https://docs.camunda.io/docs/next/apis-tools/operate-api/overview/#filter-dates.